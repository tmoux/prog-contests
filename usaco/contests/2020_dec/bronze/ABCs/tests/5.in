160813849 117233779 494318659 333504810 278047628 611552438 450738589
