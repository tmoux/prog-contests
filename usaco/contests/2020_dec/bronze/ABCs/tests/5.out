117233779 160813849 333504810
